packages/host/app/commands/check-correctness.ts

```diff
@@ -1,11 +1,33 @@
+import { service } from '@ember/service';
+
+import {
+  isCardDocumentString,
+  isCardErrorJSONAPI,
+  type CardErrorJSONAPI,
+} from '@cardstack/runtime-common';
+
 import type * as BaseCommandModule from 'https://cardstack.com/base/command';
 
 import HostBaseCommand from '../lib/host-base-command';
 
+import { waitForRealmState } from './utils';
+
+import type CardService from '../services/card-service';
+import type CommandService from '../services/command-service';
+import type RealmService from '../services/realm';
+import type StoreService from '../services/store';
+
+const CARD_INDEX_TIMEOUT_MS = 15_000;
```

Comment:
is this the amount of time to wait for indexing? our prerenderer actually has this as well, which is probably the authorative source: realm-server/prerender/utils.ts `renderTimeoutMs`. perhaps we should centralize this and you can import that?

Reply:
Yes, this is the amount of time the correctness check will wait for the invalidation event to arrive. So I think it maps to what you described. I adjusted this in https://github.com/cardstack/boxel/pull/3620/commits/74b74f85a431be29cd5bef34ca5731e157416bf9